tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -783,6 +808,12 @@ var CESIUM_MAP_VIEW = (function() {
         return true;
     }
 
+    cesium_time_callback = function (interval) {
```

Comment:
Remove space between `function` and call arguments: `cesium_time_callback = function(interval) {`